### PR TITLE
Fixing issue #3: Ignore New Tab

### PR DESCRIPTION
**Pull request by jvdub**
Fixing issue #3. I've tested this and it appears to be working. I created a new function to handle tabs you want ignored. I also added the use of `hasOwnProperty` in the `for ... in` loop for better safety and performance.

Thanks for letting me contribute!

### CODE REVIEW

**Comment by jvdub** [comment]
Just re-read the issue and found that I didn't account for duplicate tabs. I'll leave this here but I'll plan on creating a new pull request when I get some more time to address duplicate tabs.

File: popup.js [hunk 1]
```diff
@@ -42,21 +42,30 @@ document.addEventListener('DOMContentLoaded', function() {
     }
   });
 
+  function isValidTab(tab) {
+    return !(tab.url.includes('chrome://') || tab.url.includes('about:blank'));
```
Comment (dngrhm) [inline popup.js]: Could you add chrome-extension to the list?

File: popup.js [hunk 2]
```diff
@@ -42,21 +42,30 @@ document.addEventListener('DOMContentLoaded', function() {
...
+          }
+          chrome.tabs.remove(tabs[i].id);
+        }
       }
     });
   }
 
   function unload_stache() {
-    _this = this;
+    var _this = this;
```
Comment (dngrhm) [inline popup.js]: Good catch here!

File: popup.js [hunk 3]
```diff
@@ -42,21 +42,30 @@ document.addEventListener('DOMContentLoaded', function() {
...
   function load_stache(node) {
     chrome.tabs.getAllInWindow(null, function(tabs) {
       chrome.tabs.create({});
       for (var i in tabs) {
-        chrome.bookmarks.create({
-          'parentId': node.id,
-          'title': tabs[i].title,
-          'url': tabs[i].url});
-        chrome.tabs.remove(tabs[i].id);
+        if (tabs.hasOwnProperty(i)) {
```
Comment (dngrhm) [inline popup.js]: What's the purpose of the hasOwnProperty here? #chromeextensionnewbquestion

File: popup.js (hunk 1 above)
Comment (jvdub) [inline popup.js]: chrome:// will catch any chrome tabs. Extension, new tab, apps, etc. They'd all be caught by this.

Comment (jvdub) [inline popup.js]: Ah, I see what you mean now. I'll add that.

**Comment by jvdub** [comment]
I've made the requested changes. I was running into some issues with having the extra function, so now the solution is nearly the same as what was submitted in #5.

**Pull request closed without merging**